src/web/themes/custom/workbc_cdq/scss/_components.scss

```diff
@@ -1,1683 +1,3976 @@
+            background: url(../src/icons/arrow-left.svg) no-repeat;
+            background-size: 8px 13px;
+            width: 8px;
+            height: 13px;
+            transform: rotate(180deg);
+            top: 6px;
+            position: absolute;
+            left: 0;
+            @media all and (max-width: 767.98px) {
+              background-size: 5px 10px;
+              width: 5px;
+              height: 10px;
+              top: 8px;
+            }
+          }
         }
+      }
     }
+  }
+}
 
-    .quiz-questions {
-        color: #fff;
-        font-size: 14px;
-        display: flex;
-        align-items: center;
-        margin-right: 24px;
+.main-bg{
+  .overlay-content{
+    .alert-error{
+      position: absolute;
+      z-index: 1;
+      bottom: 130px;
+      left: 85px;
+      padding: 0;
+      background: transparent;
+      border: 0;
+      color: #CD1B21;
+      font-family: 'BCSans';
+      font-weight: 400;
+      font-style: normal;
+      font-size: 1.4rem;
+      &::before{
+        content: url(../src/icons/alert.svg);
+        position: relative;
+        top: 3px;
+        margin-right: 5px;
+      }
+      @media all and (max-width: 767.98px){
+        bottom: 105px;
+        left: 30px;
+        font-size: 1.3rem;
 
-        span {
-            display: inline-block;
-            width: 34px;
-            height: 25px;
-            background: url(../src/icons/list-white.svg) no-repeat;
+        &+div+div.block-system-main-block#block-workbc-cdq-content.has-prev-block {
+          padding-bottom: 250px;
         }
+      }
+      button.close{
+        display: none;
+      }
     }
+  }
 }
 
 
-.view-cdq-compare-careers {
-    
-    position: relative;
+//Learning styles
+.path-quiz .learning-styles-quiz .region-content .form-radios .form-item,
+.path-quiz .work-values-quiz .region-content .form-radios .form-item {
+  min-width: 31%;
+  @media all and (max-width: 767.98px){
+    min-width: 100%;
+  }
+}
 
-    .view-header {
-        padding-bottom: 25px;
-    }
+//Career profile page
+.page-node-type-career-profile {
+  .breadcrumb {
+    display: none;
+  }
+  .node--type-career-profile {
+    background-color: #e9ecef;
+  }
 
-    .bottom-btn {
-        position: absolute;
-        bottom: 0;
-        margin-bottom: 15px;
-    }
+  .career-bottom-notes {
+    padding-left: 12px;
+    padding-right: 12px;
+  }
+}
 
-    .btn-career-profile,
-    .btn-find-job {
-        width: 155px !important;
-        height: 51px !important;
-    }
+//login form
+.user-login-form {
+  max-width: 400px;
+}
+
+#block-workbc-cdq-branding {
+  .col-12 {
+    padding: 0 0;
+  }
+}
+
+.overlay-content {
+  .col-12 {
+    padding: 0 0;
+  }
+}
+
+#block-workbc-cdq-content {
+  margin: 0 20px;
+
+  @media all and (max-width: 767.98px){
+    margin: 0 0;
   }
+}
 
+// talk to Ray about removing the offending content.
+h2.cdq-quiz-title, .webform-submission-information {
+  display: none
+}
```

Comment:
Ray, I assume hiding this content is okay with you?

Reply:
@rjkuyvenhoven